### ISSUE TITLE
Cannot run "gatsby build" on gatsby version 2

### ISSUE THREAD

**Issue by abzeede** [mongkuen/gatsby-plugin-eslint#2]
```
Your plugins must export known APIs from their gatsby-node.js.
The following exports aren't APIs. Perhaps you made a typo or
your plugin is outdated?

See https://www.gatsbyjs.org/docs/node-apis/ for the list of Gatsby Node APIs
 — The plugin "gatsby-plugin-eslint@1.0.3" is exporting a variable named "modifyWebpackConfig" which isn't an API. Perhaps you meant to export "onCreateWebpackConfig"?
error An unexpected error occurred: "Command failed.
Exit code: 1
Command: sh
Arguments: -c gatsby build
```
environment:
- gatsby@next
- node version 10.3.0

**Comment by mongkuen** [comment]
Added some dev scripts, updated the readme, versions, and published to npm. 

Sorry about the delay. Thanks for the PR, I really appreciate it!
